seed/fastapi/plain-text/resources/service/service/service.py

```diff
@@ -67,7 +67,7 @@ def wrapper(*args: typing.Any, **kwargs: typing.Any) -> str:
 
         router.post(
             path="/text",
-            response_model=str,
+            response_class=str,
```

Comment:
know why this changed?

Reply:
Yeah will update, this is technically correct, but will revert for strings to maintain parity